src/openrct2/interface/window.h

```diff
@@ -36,6 +36,7 @@ struct TitleSequence;
 struct TextInputSession;
 
 #define TEXT_INPUT_SIZE 1024
+#define TOP_TOOLBAR_CLEARANCE 28
```

Comment:
Feel free to create a define for TOP_TOOLBAR_HEIGHT that has the value of 27 and change the existing usages of TOP_TOOLBAR_CLEARANCE to TOP_TOOLBAR_HEIGHT + 1.